tikv/pd
# test: reorganize the TSO test and reduce the integration test time cost

Signed-off-by: JmPotato <ghzpotato@gmail.com>

### What problem does this PR solve?

Resolve #3838. Reorganize the TSO test and reduce the integration test time cost.

### What is changed and how it works?

* Classify TSO tests into function tests and consistency tests.
* Only run TSO function test in a PR independently.
* Only run TSO consistency test after code is merged into the master.
* Remove some unnecessary tests.
* `make test` will only run the TSO function test.

### Check List



Tests



- Unit test
- Integration test
- Manual test
 
<img width="805" alt="图片" src="https://user-images.githubusercontent.com/1446531/126929330-3f74ac99-0599-44c1-badc-5f4eaab49e5d.png">

<img width="847" alt="图片" src="https://user-images.githubusercontent.com/1446531/126926755-435d4aba-9bd6-47db-8212-10ba78bd43b6.png">

Usually, these tests will last about 11 minutes because the TSO test will be fully run in the chunk 2. Now it can be run independently, so the overall time cost will drop to about 8 minutes, which is a 27% time-saving. However, the `idc-jenkins-ci/test` still takes a long time to run. We should make some changes to the Jenkins workflow settings later.

### Release note

```release-note
None.
```
